src/System.Diagnostics.Process/tests/ProcessTests.cs

```diff
@@ -342,10 +342,19 @@ public void TestVirtualMemorySize64()
             Assert.True(_process.VirtualMemorySize64 > 0);
         }
 
-        [ActiveIssue(3281, PlatformID.OSX)]
         [Fact]
         public void TestWorkingSet64()
         {
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.OSX))
+            {
+                // resident memory can be 0 on OSX. From apple documentation: 
+                // Wired: Information in RAM that can't be moved to the Mac's drive.
+                // The amount of Wired memory depends on the applications you are using.
+                // Does not specify if there is a significant wired memory for each app.
```

Comment:
Is this a copy of text from the docs?  Is that ok to include as-is?  Seems like it'd be sufficient to just have the first sentence "resident memory can be 0 on OS X".